charts/stable/zigbee2mqtt/docs/installation.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Installation Guide
+---
+
+### TrueNAS SCALE Installation Hints
+
+Here's an example of a default installation of zigbee2mqtt. Pay particular attention to the section "Configure Mount USB Devices" to configure the USB adapter to your container as those two paths should match. 
+
+![config1](images/config1.png)
+![config2](images/config2.png)
+![config3](images/config3.png)
+![config4](images/config4.png)
+![config5](images/config5.png)
+![config6](images/config6.png)
```

Comment:
While images are good, I think some text is needed, so people can search/index and find it

Reply:
Agree, but since this go's to the very nice overview of all charts. it is the default location to look at. with installation, truecharets and zigbee2mqqt it should be nicely indexable.